classes/submission/reviewAssignment/ReviewAssignment.inc.php

```diff
@@ -469,16 +469,19 @@ function getStatus() {
 		if ($this->getDeclined()) {
 			return REVIEW_ASSIGNMENT_STATUS_DECLINED;
 		} elseif (!$this->getDateCompleted()) {
+			$responseDueTime = strtotime($this->getDateResponseDue());
+			$reviewDueTime = strtotime($this->getDateDue());
+			$yesterdayTime = strtotime('-1 day');
```

Comment:
I believe here there should be `strtotime('today')` which is same as `strtotime('today midnight')`. This would return the midnight timestamp, i.e.: lets say today is `2018-03-27`, than it would be the timestamp of `2018-03-27 00:00:00`, which is what we are looking for, right?
As it is now: Lets say the due date is `2018-03-26 19:30:40`. `$yesterdayTime` would be the timestamp of `2018-03-26 18:00:00`, which is less than the due date (because of the hours). Thus it would not be marked as overdue.

Reply:
At the moment, we only have dates, not times, for deadlines. So if the due date is `2018-03-26 00:00:00`, we actually want it to not be marked overdue until `2018-03-27 00:00:01`.

It would probably make the logic clearer, though, if I just added 24 hours of seconds to the deadline times and left the time we're checking against at `time()`. Let me work that up real quick.

Reply:
I changed it to this: https://github.com/pkp/pkp-lib/pull/3524/commits/222dbb7f5ab43b5d884f06a0857fcc123843e0b0  Does that seem better?

Reply:
Ah, sorry @NateWr -- I saw the dates in the DB table review_assignments and they were with times (H:i:s) so I thought that those due dates are with times as well :-\ Yes, that way the logic is good -- it was also here witch "-1 day" :-\ but now it is even more clear. So yes, please go ahead... Sorry :-(